src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx

```diff
@@ -22,6 +22,8 @@ const BookingPill = styled.div<{
   color: string
   isAnalyticsView: boolean
 }>`
+  padding-left: ${'1px'};
```

Comment:
Does we need to use template placeholders`${someValue}` here?

Reply:
Yes it is safe, I went with `padding-left: '1px'` as in a regular style object as first approach and that didn't work, thats why I used the templating afterwards.